code-of-conduct.md

```diff
@@ -1,3 +1,3 @@
 # Kubernetes Community Code of Conduct
 
-Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
+Please refer to our [Kubernetes Community Code of Conduct](https://github.com/kubernetes/community/blob/master/code-of-conduct.md)
```

Comment:
both links work for me:
https://git.k8s.io/community/code-of-conduct.md
https://github.com/kubernetes/community/blob/master/code-of-conduct.md

https://git.k8s.io/community/code-of-conduct.md is just a shortcut for the other link.
is there a recommendation to not use the git.k8s.io link for some reason?

Reply:
sry ,that‘s a mistake